Dockerfile.rhel

```diff
@@ -1,11 +1,12 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
-WORKDIR /go/src/github.com/openshift/extended-platform-tests
+WORKDIR /go/src/github.com/openshift/openshift-tests
 COPY . .
 RUN make build WHAT=cmd/extended-platform-tests && \
     mkdir -p /tmp/build && \
-    cp /go/src/github.com/openshift/extended-platform-tests/extended-platform-tests /tmp/build/extended-platform-tests
+    cp /go/src/github.com/openshift/openshift-tests/extended-platform-tests /tmp/build/extended-platform-tests
 
-FROM registry.svc.ci.openshift.org/ocp/4.3:cli
+# FROM registry.svc.ci.openshift.org/ocp/4.3.0:kubernetes-tests
+FROM registry.svc.ci.openshift.org/somalley1/stable:kubernetes-tests
```

Comment:
I've been using ci ns `somalley1` to test with: registry.svc.ci.openshift.org/somalley1/4.3.0:kubernetes-tests